packages/manager/src/queries/cloudpulse/queries.ts

```diff
@@ -113,6 +114,15 @@ export const queryFactory = createQueryKeys(key, {
     switch (resourceType) {
       case 'dbaas':
         return databaseQueries.databases._ctx.all(params, filters);
+      case 'firewall':
+        return {
+          queryFn: async () => {
+            const response = await getFirewalls(params, filters);
+            return response.data;
+          },
+          queryKey: ['firewalls', params, filters],
+        };
```

Comment:
Can we use our existing query keys? 


```suggestion
        return firewallQueries.firewalls._ctx.all;
```

That query key might need to be modified to accept params and filters, but that can be done in this PR I think

Reply:
Done, updated firewalls' queryfn to accept params & filters..!

Reply:
Updated it to use `firewallQueries.firewalls._ctx.all;` only as of now as the other change requires various cypress updates as well.